cucumber/authenticators_jwt/features/authn_status_jwt.feature

```diff
@@ -899,3 +899,253 @@ Feature: JWT Authenticator - Status Check
     When I GET "/authn-jwt/raw/cucumber/status"
     Then the HTTP response status code is 500
     And the authenticator status check fails with error "CONJ00037E Missing value for resource: cucumber:variable:conjur/authn-jwt/raw/audience>"
+
+  Scenario: ONYX-10875: mapping-claims configured but secret not populated, 500 Error
```

Comment:
suppose to be ONYX-10877,  and why its not return 502?

Reply:
Acctually it should be 500 i fix the tests